astropy/coordinates/builtin_frames/fk4.py

```diff
@@ -129,16 +159,15 @@ def fk4_e_terms(equinox):
     k = 0.0056932  # in degrees (v_earth/c ~ 1e-4 rad ~ 0.0057 deg)
     k = np.radians(k)
 
+    # Explanatory Supplement to the Astronomical Almanac: P. Kenneth
+    #  Seidelmann (ed), University Science Books (1992).
+    T = (equinox.jd - jd1950) / 36525.0
     # Eccentricity of the Earth's orbit
-    e = earth.eccentricity(equinox.jd)
-
+    e = np.polyval((-0.000000126, -0.00004193, 0.01673011), T)
     # Mean longitude of perigee of the solar orbit
-    g = earth.mean_lon_of_perigee(equinox.jd)
-    g = np.radians(g)
-
+    g = np.radians(np.polyval((0.012, 1.65, 6190.67, 1015489.951), T) / 3600.0)
```

Comment:
Given the confusion below, maybe replace `np.radians` with `np.deg2rad` (they are identical, though for reasons somewhat beyond me they get separately generated from the same C source code).

Reply:
The previous version of the code used `np.radians()` and I have no reason to prefer `np.deg2rad()`.